apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console.Tests/GoogleCloudConsoleFormatterTest.cs

```diff
@@ -233,24 +208,33 @@ public void Log_TraceInformationIsLogged()
         // Start an activity.
         using var activity = new Activity("my-activity").Start();
 
-        var logEntry = new LogEntry<string>(LogLevel.Information, "LogCategory", new EventId(1), "test", exception: null, (state, exception) => state);
-        var writer = new StringWriter { NewLine = "\n" };
-
-        formatter.Write(logEntry, scopeProvider: null, writer);
-
         // JsonWriter writes boolean as JSON literal true or false.
         // We could use JsonConvert but for tests, direct literal is easy and simpler.
-        var expectedTraceSampled = activity?.Recorded ?? false ? "true" : "false";
+        var expectedTraceSampled = activity.Recorded ? "true" : "false";
 
-        var expectedJson = $"{{\"message\":\"test\",\"category\":\"LogCategory\",\"severity\":\"INFO\",\"logging.googleapis.com/trace\":\"projects/{traceProjectId}/traces/{activity?.TraceId}\",\"logging.googleapis.com/spanId\":\"{activity?.SpanId}\",\"logging.googleapis.com/trace_sampled\":{expectedTraceSampled}}}\n";
-        var actualJson = writer.ToString();
+        var expectedJson = $"{{\"message\":\"test\",\"category\":\"LogCategory\",\"severity\":\"INFO\",\"logging.googleapis.com/trace\":\"projects/{traceProjectId}/traces/{activity.TraceId}\",\"logging.googleapis.com/spanId\":\"{activity.SpanId}\",\"logging.googleapis.com/trace_sampled\":{expectedTraceSampled}}}\n";
+        var actualJson = SerializeSimpleLogEntryToJson(formatter);
 
         Assert.Equal(expectedJson, actualJson);
     }
-    
+
     private static GoogleCloudConsoleFormatter CreateFormatter(GoogleCloudConsoleFormatterOptions options = null)
     {
         options ??= new GoogleCloudConsoleFormatterOptions();
         return new GoogleCloudConsoleFormatter(options);
     }
+
+    // Logs a simple log entry to a StringWriter and returns the corresponding Json string based on specified GoogleCloudConsoleFormatter and IExternalScopeProvider.
+    private static string SerializeSimpleLogEntryToJson(GoogleCloudConsoleFormatter formatter = null, IExternalScopeProvider scopeProvider = null)
```

Comment:
Is there any place where you are not passing the formatter? If that the case, then don't make formatter an optional parameter. If we ever need it in the future we can do so then.

Or just don't pass it everywhere you are using the formatter from `CreateFormatter`.